docker-compose-tests.yaml

```diff
@@ -8,7 +8,7 @@ services:
   graphql-engine:
     image: hasura/graphql-engine:v1.0.0-beta.9
     ports:
-      - 9000:8080
+      - "${API_PORT:-8080}:8080"
```

Comment:
Here instead of enforcing the old arbitrary port `9000`, I can put `8080` as the default, which is more handy. Whatever our needs are on our CI machine should not prevent us from using the ports we want on our dev laptops.